app/src/main/AndroidManifest.xml

```diff
@@ -71,7 +71,6 @@
 
         <receiver
             android:name=".receiver.SessionScheduleReceiver"
-            android:process=":remote"
```

Comment:
I removed to get correct SharedPreferences value.
I used page below for reference.
http://stackoverflow.com/questions/10098981/sharedpreferences-in-broadcastreceiver-seems-to-not-update